source/apiVolontaria/volunteer/tests/tests_view_Events.py

```diff
@@ -594,3 +594,49 @@ def test_list_events_without_permissions(self):
             'The system failed to return some '
             'attributes : {0}'.format(attributes),
         )
+
+    def test_list_events_filter_by_date_range(self):
```

Comment:
1. This name of function is overwrite by the test just after it. 
2. `end_date` is not a valid filter in this test

I think that we never execute this function since the name is redefined just after by an other function of the same name. We should remove this one no ?

Reply:
Good catch, removed !